### PR TITLE
Grabs the ninja rework from Bay

### PR DESCRIPTION
**Pull request by ggermanicus**
See https://github.com/Baystation12/Baystation12/pull/33913 which describes this better than I can.

### CODE REVIEW

**Comment by Glloyd** [comment]
test failures on non-nerva maps are coming from the shuttle, which should be set up on a per-map basis.

File: code/game/antagonist/outsider/ninja.dm [hunk 1]
```diff
@@ -138,15 +137,15 @@ GLOBAL_DATUM_INIT(ninjas, /datum/antagonist/ninja, new)
...
+			directive += "Your interest group has recently been accused of religious insensitivity. Attempt to speak with the Chaplain and prove these accusations false."
 		if(16)
-			directive += "The Spider Clan has been bargaining with a competing prosthetics manufacturer. Try to shine [GLOB.using_map.company_name] prosthetics in a bad light."
+			directive += "Your interest group has been bargaining with a competing prosthetics manufacturer. Try to shine [GLOB.using_map.company_name] prosthetics in a bad light."
 		if(17)
-			directive += "The Spider Clan has recently begun recruiting outsiders. Consider suitable candidates and assess their behavior amongst the crew."
+			directive += "Your interest group has recently begun recruiting outsiders. Consider suitable candidates and assess their behavior amongst the crew."
 		if(18)
-			directive += "A cyborg liberation group has expressed interest in our serves. Prove the Spider Clan merciful towards law-bound synthetics."
+			directive += "A cyborg liberation group has expressed interest in our serves. Prove your interest group is merciful towards law-bound synthetics."
```
Comment (scrdest) [inline code/game/antagonist/outsider/ninja.dm:148]:
```suggestion
			directive += "A cyborg liberation group has expressed interest in our services. Prove your interest group is merciful towards law-bound synthetics."
```

File: code/modules/clothing/spacesuits/rig/suits/light.dm [hunk 2]
```diff
@@ -191,6 +192,90 @@
...
+		/obj/item/rig_module/power_sink,
+		/obj/item/rig_module/datajack,
+		/obj/item/rig_module/self_destruct,
+		/obj/item/rig_module/cooling_unit,
+		/obj/item/rig_module/maneuvering_jets
+		)
+
+/obj/item/rig/light/ninja/gcc
+	name = "gilgamesh heavy suit control module"
+	desc = "A ridiculously bulky military hardsuit with no model number or identifying logos besides a small ICCG crest on its control module. This suit is built like a tank and weighs as much as one."
```
Comment (scrdest) [inline code/modules/clothing/spacesuits/rig/suits/light.dm:218]: lore: It peeves me that you'd bother making a stealthy black-ops suit... and still leave a tiny identifying mark anywhere. Commit to committing perfidy, dangit!

**Comment by ggermanicus** [comment]
Shuttles now function and pass travis by making them different shuttles than what they are in the template, which is why the merc shuttle already worked.

**Comment by Glloyd** [comment]
as mentioned on discord, i don't love messing with this many bay dmis, but if grant is agreeing to suffer in the merge mines, then it is 👍

**Pull request merged**